olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -705,9 +729,10 @@ size_t NetworkCurl::RxFunction(void* ptr, size_t size, size_t nmemb,
         if (handle->payload->tellp() != std::streampos(handle->count)) {
           handle->payload->seekp(handle->count);
           if (handle->payload->fail()) {
-            OLP_SDK_LOG_WARNING(
-                kLogTag,
-                "Reception stream doesn't support setting write point");
+            OLP_SDK_LOG_WARNING(kLogTag,
+                                "Reception stream doesn't support setting "
```

Comment:
Too long!

Reply:
Any alternative? Note that this log is used in other Network implementations, so we have to align with them too.

Reply:
Of course we need. This is legacy code and it is about time that we do some face lifting.
"Payload seekp() failed" for example.

Reply:
Renaming this log in other network implementations will be done in corresponding commits.